treeherder/model/exchanges.py

```diff
@@ -10,17 +10,17 @@ class TreeherderPublisher(PulsePublisher):
     """
     exchange_prefix = "v1/"
 
-    resultset_action = Exchange(
-        exchange="resultset-actions",
-        title="Actions issued by resultset",
+    push_action = Exchange(
+        exchange="push-actions",
```

Comment:
we need to revert the name of the exchange.  This is another one that will require coordination with external consumers.  We can rename the variable ``push_action`` though.